src/util/shortRelativeDatetime.js

```diff
@@ -42,4 +42,8 @@ export const shortDatetime = curry((ref, date) => {
 	return momentDate.format('MMM D, YYYY')
 })
 
+export const messageDateTime = (date) => {
+	return moment(date * 1000).format(('MMMM D, YYYY H:mm'))
```

Comment:
this is a tricky topic. I think the format should depend on the user's locale. Let's go with the format `lll`. See https://momentjs.com/ for examples.